script.js

```diff
@@ -0,0 +1,11 @@
+const menu = document.querySelector('.menu');
+const closee = document.querySelector('.close');
+const navLinks = document.querySelector('.nav-links');
```

Comment:
- [ ]  Hey, here you're targetting the container of each nav link, instead of all the links, so that when a user clicks on any link, it should take them to that section of your portfolio and the menu should disappear too. Currently, this is not working as expected. Please, kindly improve on this to meet the project requirement.